spec/models/document_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+
+describe Document do
+  describe "initialization" do
+    let(:attrs) do
+      {
+        title: 'A title',
+        link: 'link.com'
+      }
+    end
+    let(:finder) do
+      double('finder', date_metadata_keys: [], text_metadata_keys: [])
+    end
+    let(:document) { described_class.new(attrs, finder) }
+
+    context 'without a public timestamp' do
+      it 'defaults to nil' do
+        expect(document.public_timestamp).to be_nil
```

Comment:
Can we inline some of the `let`s here? I think it would be good to show that the rummager document doesn't have a `public_timestamp` inside the `it` block.